static/src/components/GameScreen.js

```diff
@@ -66,59 +67,76 @@ export class GameScreen extends Component {
+                                    return (
+                                        <Card key={pcard.id} style={{ display: 'inline-block' }}>
+                                            <CardTitle title={pcard.card.name} />
+                                            <CardMedia>
+                                                <img alt="" src={`dist/images/cards/${imageName}.png`} />
+                                            </CardMedia>
+                                        </Card>
+                                    );
+                                })
+                            }
+                            {game.cards &&
+                                game.cards.map((card) => {
+                                    const imageName = (card.name).replace(/\s+/g, '').toLowerCase();
+                                    return (
+                                        <Card key={card.id} style={{ display: 'inline-block' }}>
+                                            <CardTitle title={card.name} />
+                                            <CardMedia>
+                                                <img alt="" src={`dist/images/cards/${imageName}.png`} />
+                                            </CardMedia>
+                                            <CardText>
+                                                <p>Discription of what card does</p>
+                                            </CardText>
+                                            <CardActions>
+                                                <FlatButton
+                                                    label="Play Card"
+                                                    onClick={() => this.playCard(card.id)}
+                                                />
+                                                <FlatButton
+                                                    label="Discard"
+                                                    onClick={() => this.discard(card.id)}
+                                                />
+                                            </CardActions>
+                                        </Card>
+                                    );
+                                })
+                            }
+                        </div>
+                    </div>
+                }
+                <div>
+                    <div style={{ float: "left", padding: 100 }}>
+                        {!error && !game && !started &&
+                            <RaisedButton
+                                label="I am ready"
+                                onClick={() => this.startGame()}
+                            />
+                        }
+                        {!error && !game &&
+                            <div style={{ padding: '50' }} >
+
+                              <p>Waiting on more players... <i id="number">#(playersLoggedIn)</i> players so far.</p>
```

Comment:
Where is playersLoggedIn initialised?

Reply:
It is not...
I have been trying to get the number of players logged in the same game at the that particular point in time, but nothing is ever returned in the playerCount prop in this file. So I have just placed a string "#(playersLoggedIn)" to remind me to look into this at some point.
- This will give the user an idea how many more players needed to start a game.

Reply:
Yeap i think this is a bug in my reducer for the game state.